providers/catalog.go

```diff
@@ -161,12 +165,15 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 	// Outreach configuration
 	Outreach: {
 		AuthType: Oauth2,
-		BaseURL:  "https://api.outreach.io",
+		BaseURL:  "https://api.outreach.io/api",
```

Comment:
I don't think we should even have `api` in this - there could be other modules that they introduce which could affect this

Reply:
Ok